.pfnci/schema.yaml

```
@@ -225,6 +230,8 @@ cudnn:
         alias: "11.6"
       "11.6":
         alias: "11.6"
+      "11.7":
+        alias: "11.7"
```

https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/
```
libcudnn8_8.4.0.27-1+cuda10.2_amd64.deb 224MB 2022-04-01 02:25
libcudnn8_8.4.0.27-1+cuda11.6_amd64.deb 402MB 2022-04-01 02:21
libcudnn8_8.4.1.50-1+cuda10.2_amd64.deb 225MB 2022-05-22 20:26
libcudnn8_8.4.1.50-1+cuda11.6_amd64.deb 401MB 2022-05-20 22:34
```

```suggestion
        alias: "11.6"
```